hydra-core/lib/generators/hydra/templates/ability.rb

```diff
@@ -1,4 +1,5 @@
 class Ability
+  include Blacklight::AccessControls::Ability
```

Comment:
Ah, thanks.  I had moved that include statement, but I forgot about the generator.  I'll fix it.